packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Send/SelectAddress/Recents/index.tsx

```diff
@@ -36,36 +64,49 @@ const Label = styled(Text)`
 
 const Recents: React.FC<Props> = (props) => {
   const {
-    data: { addresses }
+    data: { addresses },
+    ethTransactionsActions,
+    selectSearchedAddress
   } = props
 
+  useEffect(() => {
+    ethTransactionsActions.initialized()
+  }, [])
+
   return (
     <Wrapper>
       <SubTitle size='16px' lineHeight='21px' weight={500}>
         <FormattedMessage id='plugin.send.recents_title' defaultMessage='Recents' />
       </SubTitle>
-      {addresses.map((address: string) => (
-        <WallerWrapper key={address}>
-          <Icon size='24px' name={AvailableCoins.ETH} />
-          <Label size='16px' lineHeight='21px' weight={600} color='white'>
-            <CryptoAddress>{address}</CryptoAddress>
-          </Label>
-        </WallerWrapper>
-      ))}
+      <Wallets>
+        {addresses.map((address: string) => (
```

Comment:
can we have undefined here?

Reply:
@TarasPotsikaylo i have added condition for empty array in selector